source-local/src/androidMain/kotlin/tachiyomi/source/local/LocalSource.kt

```diff
@@ -206,13 +206,24 @@ actual class LocalSource(
         return@withIOContext manga
     }
 
-    private fun copyComicInfoFileFromArchive(chapterArchives: List<UniFile>, folder: UniFile): UniFile? {
+    private fun <R> getComicInfoFileFromChapter(chapterArchive: UniFile, block: (InputStream) -> R): R? {
```

Comment:
```suggestion
    private fun <T> getComicInfoForChapter(chapter: UniFile, block: (InputStream) -> T): T? {
```
For generic anything works but I prefer `T` in this kind of context